Tests/FluentAssertions.Extensibility.Specs/FluentAssertions.Extensibility.Specs.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net47</TargetFrameworks>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\Src\FluentAssertions\FluentAssertions.snk</AssemblyOriginatorKeyFile>
+    <IsPackable>false</IsPackable>
+    <NoWarn>$(NoWarn),IDE0052,1573,1591,1712</NoWarn>
+    <DebugType>full</DebugType>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net47'">
+    <!--
+      (SRCU = System.Runtime.CompilerServices.Unsafe)
+      FluentAssertions.csproj depends on SRCU 4.5.0 (AssemblyVersion 4.0.4.0)
+      System.Collections.Immutable 1.6.0+ depends on SRCU 4.5.2+ (AssemblyVersion 4.0.4.1)
+      FluentAssertions.Specs.csproj for net47 does not generate a binding redirect from 4.0.4.0 to 4.0.4.1
+      Upgrading to 1.6.0 gives "Could not load file or assembly 'System.Runtime.CompilerServices.Unsafe, Version=4.0.4.0"
+    -->
+    <PackageReference Include="System.Collections.Immutable" Version="[1.5.0]" />
```

Comment:
This can probably be removed.
As I recall it's only there to get access to `ImmutableArray`.